sale_commission_product_criteria_country/models/commission_line_mixin.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2025 Alberto Martínez <alberto.martinez@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class CommissionLineMixin(models.AbstractModel):
+    _inherit = "commission.line.mixin"
+
+    def _commission_items_query_params(self, commission, product):
+        res = super()._commission_items_query_params(commission, product)
+        sale_commission_customer = self.env.context.get("sale_commission_customer")
+        if sale_commission_customer:
+            res["country"] = (
+                sale_commission_customer.country_id.id
+                if sale_commission_customer.country_id
+                else 0
+            )
+        return res
+
+    def _commission_items_where(self):
+        res = super()._commission_items_where()
+        if self.env.context.get("sale_commission_customer"):
+            res = (
+                res
+                + """ AND (
```

Comment:
Don't you think you could use an f function to concatenate strings?

Reply:
I can, but i think its not important here because we are doing a common concatenation

Reply:
It's OK with me, but what do you think @ValentinVinagre

Reply:
@Tisho99 change it please.

Reply:
Changed